merkledag/batch.go

```diff
@@ -0,0 +1,98 @@
+package merkledag
+
+import (
+	"runtime"
+
+	cid "gx/ipfs/QmNp85zy9RLrQ5oQD4hPyS39ezrrXpcaa7R4Y9kxdWQLLQ/go-cid"
+	node "gx/ipfs/QmPN7cwmpcc4DWXb4KTB9dNAJgjuPY69h3npsMfhRrQL9c/go-ipld-format"
+	blocks "gx/ipfs/QmSn9Td7xgxm9EV7iEjTckpUWmWApggzPxu7eFGWkkpwin/go-block-format"
+)
+
+// ParallelBatchCommits is the number of batch commits that can be in-flight before blocking.
+// TODO: Experiment with multiple datastores, storage devices, and CPUs to find
```

Comment:
Can you create issues for this instead of in code TODO (they just get forgotten).

Reply:
Fine! (...grumble... we'll never get to it anyways)

Reply:
that might be true but it is still better than having TODO in code.
As you said in your issue, someone might just take a stab at it out of pure boredom.

Reply:
You're right, I was just being lazy 🙂.